content/content-strategy/define-user-content-needs/create-actionable-artefacts/user-job-stories/index.yml

```diff
@@ -0,0 +1,27 @@
+pagetitle: User and job stories
+description: How to write user stories to create digital content.
+weight: 10
+layout: layout/topic
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
+  - intro.md #methods
+  - toc.md
+  - supporting-takeaway.md
+  - heading1.md
+  - text.md
+  - heading2.md
+  - text1.md
+  - heading3.md
+  - section.md
+  - diagram.md #diagram
+  - blanktextspacer.md #quickfix
+  - blanktextspacer.md #quickfix
```

Comment:
@sirleech Will this be addressed in some of the UI-Kit 2 conversion, or does the spacing issue need to be reported as a separate thing? Just want to ensure we capture the issue elsewhere. cc @Phend-117

Reply:
@sirleech please confirm.

Reply:
PLEASE CHECK